stdlib/public/Darwin/Accelerate/vDSP_FillClearGenerate.swift

```diff
@@ -731,5 +731,275 @@ extension vDSP {
             }
     }
     
+    //===----------------------------------------------------------------------===//
+    //  Generation by extrapolation and interpolation.
+    //===----------------------------------------------------------------------===//
+    
+    /// Fills a single-precision vector by extrapolation and interpolation.
+    ///
+    /// - Parameter values: The values to interpolate, for example, intensities, magnitudes, or some other measurable quantities.
+    /// - Parameter indices: The monotonically increasing indices to `result` for the corresponding element in `values`.
+    /// - Parameter result: Output values.
+    @inlinable
+    @available(macOS 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, *)
```

Comment:
These need to be changed to `@available(macOS 9999, iOS 9999, tvOS 9999, watchOS 9999, *)` everywhere, because we don't know which OS versions will ship these yet.

Reply:
Hi, this is fixed now.